eox_nelp/middleware.py

```diff
@@ -93,18 +110,19 @@ def __call__(self, request):
 
         return self.get_response(request)
 
-    def _generate_handler(self, field):
+    def _generate_handler(self, field, label=None):
         """Every field will require a handler method, check the logic here
         https://github.com/eduNEXT/edunext-platform/blob/ednx-release/mango.master.nelp/
         openedx/core/djangoapps/user_authn/views/registration_form.py#L360,
         this method generates a specific handler for each new field with its specific label.
 
         Args:
             field: String
+            label: String or None
         Returns:
             func: Handler method
         """
-        label = field.capitalize()
+        label = label.capitalize()
```

Comment:
If label is None ? this could happen??
![image](https://user-images.githubusercontent.com/51926076/204350992-d953480c-fa58-4d3a-80b1-af9b1db001dc.png)

Reply:
with the current implementation that can not happen however I will remove the default value since that comes from a previous implementation

Reply:
I have a question, if you remove the default value. Why this method `_generate_handler `https://github.com/eduNEXT/eox-nelp/pull/13/files#diff-66a15828789bf584965751e47deaa844139ec753886961cc4c1450c06e733c70R113
uses two argument if only use the label. ?